lib/routes/spglobal/ratings.ts

```diff
@@ -0,0 +1,78 @@
+import { Route, ViewType } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import got from '@/utils/got';
+
+export const route: Route = {
+    path: '/ratings/:language?',
+    categories: ['finance', 'popular'],
+    view: ViewType.Articles,
+    example: '/ratings/en/',
+    parameters: {
+        language: {
+            description: '语言',
+            options: [
+                { value: 'zh', label: '中文' },
+                { value: 'en', label: 'English' },
+                { value: 'es', label: 'Español' },
+                { value: 'pt', label: 'Português' },
+                { value: 'jp', label: '日本語' },
+                { value: 'ru', label: 'Русский' },
+                { value: 'ar', label: 'العربية' },
+            ],
+        },
+    },
+    features: {
+        requireConfig: [
+            {
+                name: 'LANGUAGE',
+                description: '语言',
+            },
+        ],
```

Comment:
`requireConfig` defines environment variables that need to explicitly set if a user wants to subscribe to this route and I don't see any usage of the environment variables `LANGUAGE`